rpm/qmllive.spec

```diff
@@ -46,6 +46,7 @@ make install INSTALL_ROOT=%{buildroot}
 
 %files
 %defattr(-,root,root,-)
+%license LICENSE.GPL3
```

Comment:
IIRC this macro breaks shadow building with mb2/sfdk, which is also what Qt Creator does by default... I hope we are able to fix it at some point.